src/main/groovy/org/jenkinsci/gradle/plugins/jpi/GenerateTestHpl.groovy

```diff
@@ -0,0 +1,20 @@
+package org.jenkinsci.gradle.plugins.jpi
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.file.DirectoryProperty
+import org.gradle.api.tasks.OutputFile
+import org.gradle.api.tasks.TaskAction
+
+class GenerateTestHpl extends DefaultTask {
+    @OutputFile
```

Comment:
Shouldn't this be `@OutputDirectory`?

I get a `org.gradle.api.InvalidUserDataException: Cannot write to file '...\job-dsl-plugin\job-dsl-plugin\build\generated-resources\test' specified for property 'hplDir' as it is a directory.` when testing the change with Job DSL.